docs: improve operations documentation (#280)
Substra/substra-documentation: docs/source/operations/walkthrough/30-backend-deployment.rst

******************
Deploy the backend
******************

> Here we will deploy a node for the ``ingen`` organization onto ``cluster-1``.

Reviewer: ```suggestion
In this step we deploy a node for the ``ingen`` organization onto ``cluster-1``.
```

> Obviously this will need to be repeated for ``biotechnica`` onto ``cluster-2``, with the appropriate values changed.

Reviewer: ```suggestion
This needs to be repeated for ``biotechnica`` onto ``cluster-2``, with the appropriate values changed.
```
